go/obscurocommon/common.go

```diff
@@ -23,6 +23,6 @@ type Nonce = uint64
 type EncodedRollup []byte
 
 type NotifyNewBlock interface {
```

Comment:
this interface is not needed any more.
There is no point in having an interface in the main code to be used only in an integration test.
I propose we remove it

Reply:
Moved it to the ethereum mock package 👍